spec/logic_spec.rb

```diff
@@ -0,0 +1,60 @@
+# spec/logic_spec.rb
+
+require './lib/logic'
+
+# rubocop:disable Metrics/BlockLength
+
+describe Player do
+  describe '#game_name' do
+    it 'returns the name of game' do
+      my_game = Player.new
+      expect(my_game.game_name).to eql('Welcome to TIC TAC TOE game')
+    end
+  end
+
+  describe '#sign_validate' do
+    it 'returns boolean state of players' do
+      sign_validated = Player.new
+      expect(sign_validated.sign_validate('x')).to eql(true)
+      expect(sign_validated.sign_validate('o')).to eql(true)
```

Comment:
- [ ] You should have one "it" block per test expectation. Here, you're using one "it" block for multiple test expectations.

Reply:
@george-shammar you mentioned that `spec/logic_spec.rb is outdated`. Please what is its current form?

Reply:
That's an auto message from github